keyboards/xelus/valor/rev3/info.json

```diff
@@ -0,0 +1,122 @@
+{
+    "keyboard_name": "Valor Rev3",
+    "manufacturer": "Xelus",
+    "url": "",
+    "maintainer": "Xelus22",
+    "processor": "atmega32u4",
+    "bootloader": "atmel-dfu",
+    "usb": {
+        "vid": "0x5845",
+        "pid": "0x5652",
+        "device_version": "0.0.1"
+    },
+    "matrix_pins": {
+        "rows": ["B1", "B2", "C7", "C6", "B6"],
+        "cols": ["E6", "F0", "F1", "F4", "F5", "F6", "F7", "B5", "B4", "D7", "D6", "D4", "D5", "D3", "D2"]
+    },
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true
+    },
+    "rgblight": {
+        "led_count": 32,
+        "pin": "B0",
```

Comment:
This will need to be migrated to ws2812.pin on develop (no action needed, currently)